git_machete/github.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+import http.client
+import json
+import os
+import re
+from http.client import HTTPResponse, HTTPSConnection
```

Comment:
it belongs to std lib... cool :rocket:

Reply:
Actually... that's the bad client, the good one is apparently `requests` (https://requests.readthedocs.io/en/master/)... feels so much easier to use...

Reply:
But `requests` don't belong to std lib, and I don't want to add any external deps unless absolutely necessary (so far it hasn't been, and let it stay this way)